src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -445,8 +447,8 @@ private synchronized void onUserJoinedChannel(ChatUser chatUser) {
       if (!userSearchTextField.textProperty().get().isEmpty()) {
         chatUserItemController.setVisible(isUsernameMatch(chatUserItemController));
       }
-      if (filterUserPopup.isShowing()) {
-        userFilterController.filterUser(chatUserItemController);
+      if (userFilterController.isFilterAppliedProperty().get()) {
```

Comment:
This somewhat doubled, instead I'd add a isFilterApplied() getter to UserFilterController. (additionally to the property getter)